test/nuts/mdapi.nut.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { expect } from 'chai';
+
+import { TestSession, execCmd } from '@salesforce/cli-plugins-testkit';
+
+let session: TestSession;
+
+describe('mdapi NUTs', () => {
```

Comment:
again same worry. I think mdapi NUTs will mirror source NUTs

Reply:
They might but I'm hoping they can be covered by 1 NUT file.  If I'm wrong I'll change it.

Reply:
so are we assuming that if source NUTs pass, that identical mdapi NUTs would pass too?

Reply:
No, that's not the assumption.  mdapi commands will have NUTs added as they are added to the plugin.

Reply:
But why don't we need as many mdapi NUTs as we have source NUTs then?